LiteFarmOrg/LiteFarm
# LF-2376/notify user when task reassigned to someone else

Ticket [LF-2376](https://lite-farm.atlassian.net/browse/LF-2376)

**Description**
We want to notify a user when their task has been reassigned by assigner (i.e. farm owner or manager) to someone else.

**To Test:**
1. You will need at least one worker at your farm that you have access to.
2. Create a task and assign it to the worker from step 1.
3. Reassign the same task to someone else.
4. Log in to the account of the worker you initially assigned the task to from step 1.
5. You will see two notifications (initial assignment notification and reassignment notification)

I use the currently authenticated user's id as the assigner from request headers.
